docs/initial_setup.md

```diff
@@ -9,7 +9,8 @@
 
 For best results, prior to using Precaution, it is recommended to run static code analysis tooling on the code base. That way the current code base does not contain known security issues.
 
-The table below lists the languages supported by Precaution and the static analysis tools used by Precaution in support of the language. You should use the appropriate tool from the list for your initial scans before enabling Precaution.
+You should use the appropriate tool from the list for your initial scans before enabling Precaution.
+The table below lists the languages supported by Precaution and the static analysis tools used by Precaution in support of the language.
```

Comment:
Not sure why this sentence order was flipped? It feels a bit less natural this way around.

Reply:
I don't remember why I did it but you are right.